modules/virt-creating-vddk-image.adoc

```diff
@@ -3,33 +3,32 @@
 // * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
 
 [id="virt-creating-vddk-image_{context}"]
-= Creating a VDDK image
+= Creating and using a VDDK image
 
-You can convert the VMware Virtual Disk Development Kit (VDDK) into an image, and push the image to an image registry. Then, add the image to the `v2v-vmware` ConfigMap.
+You can download the VMware Virtual Disk Development Kit (VDDK), build a VDDK image, and push the VDDK image to your image registry. You then add the VDDK image to the `v2v-vmware` ConfigMap.
 
 .Prerequisites
 
-* You must configure an image registry.
-* You must download the VDDK archive file.
+* You must have access to an {product-title} internal image registry or a secure external registry.
 
 .Procedure
 
+. In a browser, navigate to link:https://www.vmware.com/support/pubs/[VMware Documentation].
+. Click *VMware SDK & API Product Documentation* -> *VMware Virtual Disk Development Kit (VDDK)*.
+. Select the latest VDDK release and click *Download SDKs*.
+
 . Create and navigate to a temporary directory:
```

Comment:
This seems like it's missing a step or two. Don't you need to say to save the SDKs to a location on your file system? and then move it to this temp directory you're creating?

Or actually I think it made sense creating the temp directory first as it originally was, and then telling them to save the file into this directory.

Reply:
Yes, I think you're right.